tests/netuit/topology/scripts/compare_graphs.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+
+import networkx as nx
+import sys
+
```

Comment:
can you add a comment to these python scripts with a one sentence summary and a listing of expected command-line arguments?